src/Orleans/Core/ClientBuilder.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.DependencyInjection;
+using Orleans.CodeGeneration;
+using Orleans.Messaging;
+using Orleans.Providers;
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using Orleans.Serialization;
+using Orleans.Streams;
+
+namespace Orleans
```

Comment:
To me, Configuration is for specifying configuration rather than consuming it, that's all. I don't feel strongly about it. I felt that putting it in the core namespace made it more obvious as a starting point for users.